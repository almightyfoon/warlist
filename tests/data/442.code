c5501b_-0hb-1g2w2w2w2w2w2w2wd_9v3y3y3y4L4L3i___-0AdU1g1gc43y3y3W3X3X3-3m3o3m3o