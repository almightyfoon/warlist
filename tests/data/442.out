https://conflictchamber.com/#c5501b_-0hb-1g2w2w2w2w2w2w2wms9v3y3y3y4L4Lmg___-0AdU1g1gmx3y3y3W3X3X3-3m3o3m3o

Retribution Army - 71 / 75 points
[Theme] Forges of War

[Vyros 2] Vyros, Incissar of the Dawnguard [+27]
 - Chimera [8]
 - Griffon [8]
 - Griffon [8]
 - Griffon [8]
 - Griffon [8]
 - Griffon [8]
 - Griffon [8]
 - Griffon [8]
 - Imperatus [20]
 - Sylys Wyshnalyrr, the Seeker [0(4)]
Arcanist Mechanik [2]
Arcanist Mechanik [2]
Arcanist Mechanik [2]
House Shyeel Artificer [0(5)]
House Shyeel Artificer [0(5)]
House Shyeel Battle Mages [8]

Retribution Army - 71 / 75 points
[Theme] Shadows of the Retribution

[Issyria 1] Issyria, Sibyl of Dawn [+29]
 - Chimera [8]
 - Chimera [8]
 - Hyperion [32]
Arcanist Mechanik [2]
Arcanist Mechanik [2]
Eiryss, Mage Hunter of Ios [0(7)]
Mage Hunter Assassin [4]
Mage Hunter Assassin [4]
Narn, Mage Hunter of Ios [0(6)]
Mage Hunter Strike Force (max) [16]
 - Mage Hunter Strike Force Commander [4]
Mage Hunter Strike Force (max) [16]
 - Mage Hunter Strike Force Commander [4]
